plugin.video.carnet-meduza/resources/lib/carnet-meduza.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+import os, sys
+import urllib, urlparse, json
+import xbmcgui, xbmcplugin, xbmcaddon
+import requests
+from register import device
```

Comment:
Well every time carent-meduza.py script is loaded (from register import device) it'll check if device is properly register with service (see device.py from line 158). That's not right way of doing it?